[2025-04-23T17:27:01.802+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-04-23T17:27:01.821+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: preprocessing_pipeline.run_preprocessing manual__2025-04-23T17:26:59.384459+00:00 [queued]>
[2025-04-23T17:27:01.827+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: preprocessing_pipeline.run_preprocessing manual__2025-04-23T17:26:59.384459+00:00 [queued]>
[2025-04-23T17:27:01.828+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2025-04-23T17:27:01.837+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): run_preprocessing> on 2025-04-23 17:26:59.384459+00:00
[2025-04-23T17:27:01.847+0000] {warnings.py:110} WARNING - /home/airflow/.local/lib/python3.12/site-packages/airflow/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=172) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-04-23T17:27:01.849+0000] {standard_task_runner.py:63} INFO - Started process 174 to run task
[2025-04-23T17:27:01.850+0000] {standard_task_runner.py:90} INFO - Running: ['airflow', 'tasks', 'run', 'preprocessing_pipeline', 'run_preprocessing', 'manual__2025-04-23T17:26:59.384459+00:00', '--job-id', '4', '--raw', '--subdir', 'DAGS_FOLDER/dag_preprocessing.py', '--cfg-path', '/tmp/tmpyruufphl']
[2025-04-23T17:27:01.851+0000] {standard_task_runner.py:91} INFO - Job 4: Subtask run_preprocessing
[2025-04-23T17:27:01.883+0000] {task_command.py:426} INFO - Running <TaskInstance: preprocessing_pipeline.run_preprocessing manual__2025-04-23T17:26:59.384459+00:00 [running]> on host 7cc2314952d5
[2025-04-23T17:27:01.927+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='preprocessing_pipeline' AIRFLOW_CTX_TASK_ID='run_preprocessing' AIRFLOW_CTX_EXECUTION_DATE='2025-04-23T17:26:59.384459+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-04-23T17:26:59.384459+00:00'
[2025-04-23T17:27:01.929+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-04-23T17:27:01.929+0000] {dag_preprocessing.py:34} INFO - === INIZIO PREPROCESSAMENTO ===
[2025-04-23T17:27:01.929+0000] {dag_preprocessing.py:23} INFO - Download risorse NLTK...
[2025-04-23T17:27:02.495+0000] {dag_preprocessing.py:27} INFO - ✓ 'punkt' scaricato.
[2025-04-23T17:27:02.497+0000] {dag_preprocessing.py:27} INFO - ✓ 'stopwords' scaricato.
[2025-04-23T17:27:02.537+0000] {dag_preprocessing.py:27} INFO - ✓ 'wordnet' scaricato.
[2025-04-23T17:27:02.541+0000] {dag_preprocessing.py:44} ERROR - Errore caricamento CSV: [Errno 2] No such file or directory: "'/opt/airflow/***/sentiment_data/raw/all-data.csv"
[2025-04-23T17:27:02.542+0000] {dag_preprocessing.py:82} ERROR - ✗ Errore nel task di preprocessing: [Errno 2] No such file or directory: "'/opt/airflow/***/sentiment_data/raw/all-data.csv"
[2025-04-23T17:27:02.542+0000] {python.py:237} INFO - Done. Returned value was: None
[2025-04-23T17:27:02.543+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-04-23T17:27:02.553+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=preprocessing_pipeline, task_id=run_preprocessing, run_id=manual__2025-04-23T17:26:59.384459+00:00, execution_date=20250423T172659, start_date=20250423T172701, end_date=20250423T172702
[2025-04-23T17:27:02.576+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-04-23T17:27:02.593+0000] {taskinstance.py:3498} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-04-23T17:27:02.594+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
